config/csm/csm_config.yaml

```diff
@@ -25,8 +25,8 @@ Restcall:
   s3accounts_endpoint : "/api/v2/s3_accounts"
   s3_bucket_endpoint : "/api/v2/s3/bucket"
   IAM_users_endpoint : "/api/v2/iam_users"
-  s3_iam_user_endpoint : "/api/v2/s3/iam/users"
-  csmuser_endpoint : "/api/v2/csm/users"
+  s3_iam_user_endpoint : "/api/v2/iam/users"
+  csmuser_endpoint : "/api/v2/system/users"
```

Comment:
This is for LR so we can skip the change from here

Reply:
updated